src/server/internalServer.cpp

```diff
@@ -759,10 +759,22 @@ std::unique_ptr<Response> InternalServer::handle_viewer_settings(const RequestCo
   return ContentResponse::build(*this, RESOURCE::templates::viewer_settings_js, data, "application/javascript; charset=utf-8");
 }
 
+std::string InternalServer::dumpDownloadLinks(const std::string& bookId) const
```

Comment:
In the content of `InternalServer` the verb `dump` sounds a little strange. `getNoJSDownloadPageHTML()` is a better name.